xonsh/ptk/shortcuts.py

```diff
@@ -64,6 +64,8 @@ def prompt(self, message='', **kwargs):
 
         # Create CommandLineInterface.
         if self.cli is None:
+            if self.major_minor < (0, 57):
+                kwargs.pop('reserve_space_for_menu', None)
```

Comment:
I think that this should probably still be merged, even if there is no response to the other issue.  This is still fixes a bug.